src/js/components/DataChart/__tests__/__snapshots__/DataChart-test.js.snap

```diff
@@ -6971,6 +6971,34 @@ exports[`DataChart pad 1`] = `
 </div>
 `;
 
+exports[`DataChart renderrs a11yTitle and aria-label 1`] = `
+.c0 {
+  font-size: 18px;
+  line-height: 24px;
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+}
+
+.c1 {
+  position: relative;
+  grid-area: charts;
+}
+
+<div
+  class="c0"
+>
+  <div
```

Comment:
Where's the `aria-label`?

Reply:
hmmm seems like something isn't applying correctly to DataChart. maybe we can look at that separately from this PR.

Reply:
If we remove DataChart from this PR, I'm OK with separating it. I'd rather not go forward with a snapshot we know isn't right.